File: Sort_Colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zeros = 0
        ones = 0
        twos = 0
        for i in nums:
            if i == 0:
                zeros+=1
            elif i == 1:
                ones+=1
            else:
                twos+=1
        for i in range(0,zeros):
            nums[i] = 0
        for i in range(zeros,zeros+ones):
            nums[i] = 1
        for i in range(zeros+ones,zeros+ones+twos):
            nums[i] = 2
        return nums

File: sortColors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zeros= 0;
        int ones = 0;
        int twos = 0;
        for(auto num : nums){
            switch(num){
                case 0:
                    zeros++;
                    break;
                case 1:
                    ones++;
                    break;
                case 2:
                    twos++;
                    break;
            }
        }
        cout<<zeros<<ones<<twos;
        fill(nums.begin(), nums.begin() + zeros, 0);
        fill(nums.begin() + zeros, nums.begin() + zeros + ones, 1);
        fill(nums.begin() + zeros + ones, nums.end(), 2);
    }
}; 
